Commands/Utility.cs

```diff
@@ -316,5 +324,23 @@ string formatMinutesToString(int minutes)
 
             return format;
         }
+
+        [SlashCommand("sauce", "Searches for a kind of sauce.")]
+        public async Task SearchSauce(string sauceSearchText = "")
```

Comment:
NIT: to better align with the other updates I've been making, this variable would probably be called sauce